workers/python/pyproject.toml

```diff
@@ -1,5 +1,9 @@
 [tool.poetry]
-package-mode = false
+name = "python-worker"
+version = "0.0.1"
+description = "Bee API Python Worker for file extraction."
+authors = ["IBM Corp."]
+license = "Apache-2.0"
```

Comment:
Opting out from the package mode is intentional, rest of the fields can be omitted when in that mode. See https://python-poetry.org/docs/pyproject/

If you have problems with installation, check your poetry version, I think I had similar issue once.

Reply:
I got it; I was using an outdated Poetry version (1.7.1). I upgraded to 1.8.5 (latest), and it started working.